module/TaskManagement/test/integration/TaskManagement/LastSharesAssignmentProcessTest.php

```diff
@@ -103,8 +103,9 @@ public function testAssignSharesAsLast() {
 
 		$readModelTask = $this->controller->getTaskService()->findTask($this->task->getId());
 		$this->assertEquals(201, $response->getStatusCode());
-		$this->assertEquals(Task::STATUS_CLOSED, $this->task->getStatus());
-		$this->assertEquals(Task::STATUS_CLOSED, $readModelTask->getStatus());
+		$this->assertNotEquals(Task::STATUS_CLOSED, $this->task->getStatus());
```

Comment:
Direi che opportuno testare lo stato in cui deve essere e non un'altro stato a caso...